### ISSUE TITLE
why train loss will be nan?

### ISSUE THREAD

**Issue by GaoTiaoKang** [CharlieDinh/pFedMe#11]
why train loss will be nan?

**Comment by CharlieDinh** [comment]
Hi, the training loss is nan when the learning rate is large.
